example/index.js

```diff
@@ -27,7 +27,9 @@ const camera = createCamera({
 createOrbiter({
   camera: camera,
   element: ctx.gl.canvas,
-  easing: 0.1
+  easing: 0.1,
+  minDistance: 3,
```

Comment:
is that arbitrary and project specific? Most my cameras are positioned at 1,1,1...